setup.sh

```diff
@@ -62,6 +62,10 @@ function updateenv() {
     if [[ $REPLY =~ ^[Yy]$ ]]
     then
         REQUIREMENTS=requirements-dev.txt
+        REQUIREMENTS_PLOT="-r requirements-plot.txt"
```

Comment:
requirements-dev.txt actually includes all other dependencies via `-r requirements-xxx.txt` already - so if you run only `pip install requirements-dev.txt` - you'll get ALL dependencies (from all requirements files) except the doc ones - which are irrelevant.